pkg/standalone/run.go

```diff
@@ -37,8 +37,15 @@ const (
 	defaultStructTagKey  = "default"
 )
 
+// EnvItems represents the env configuration options that are present in commmon and/or individual app's section.
+type EnvItems struct {
+	Name  string `yaml:"name"`
+	Value string `yaml:"value"`
+}
+
```

Comment:
Discussed with Pravin to change this to a simple `map[string]string`, which will (a) simplify the YAML, (b) make the merging b/w common and app environment variables much simpler.